### PR TITLE
Update GrapheneOS

### PR DESCRIPTION
**Pull request by d4rklynk**
Prior to my old PR #1688 : 
- I just added some links

I did **not** add the new feature of GOS

### CODE REVIEW

File: docs/android.en.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ We recommend installing one of these custom Android operating systems on your de
...
+GrapheneOS supports [Sandboxed Google Play](https://grapheneos.org/usage#sandboxed-google-play), which runs [Google Play Services](https://en.wikipedia.org/wiki/Google_Play_Services) fully sandboxed like any other regular app. This means you can take advantage of most Google Play Services, such as [push notifications](https://firebase.google.com/docs/cloud-messaging/), while giving you full control over their permissions and access, and while containing them to a specific [work profile](https://developer.android.com/work/managed-profiles) (through [Shelter](#shelter) for instance) or [user profile](android/overview.md#user-profiles) of your choice.
```
Comment (jonaharagon) [inline docs/android.en.md:36]:
```suggestion
GrapheneOS supports [Sandboxed Google Play](https://grapheneos.org/usage#sandboxed-google-play), which runs [Google Play Services](https://en.wikipedia.org/wiki/Google_Play_Services) fully sandboxed like any other regular app. This means you can take advantage of most Google Play Services, such as [push notifications](https://firebase.google.com/docs/cloud-messaging/), while giving you full control over their permissions and access, and while containing them to a specific [work profile](android/overview.md#work-profile) or [user profile](android/overview.md#user-profiles) of your choice.
```
Makes more sense tbh. We need to be reducing the number of external links we have on the site as a long-term goal, not increasing them.

**Review by jonaharagon — CHANGES_REQUESTED** [review]

**Comment by TrashPandaCodingGarbage** [comment]
A13 stable is already pushed by Graphene

**Comment by jonaharagon** [comment]
What does that have to do with this PR?

**Comment by d4rklynk** [comment]
> A13 stable is already pushed by Graphene

wut

**Review by mfwmyfacewhen — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Pull request merged**